draft-ietf-quic-transport.md

```diff
@@ -196,6 +196,12 @@ Ack-eliciting Packet:
   CONNECTION_CLOSE. These cause a recipient to send an acknowledgment (see
   {{sending-acknowledgements}}).
 
+Out-of-order Packets:
+
+: Packets that do not increase the largest received packet number for its
+  packet number space by exactly one. Packets arrive out of order
+  when earlier packets are lost or delayed.
```

Comment:
Do we need to point out that an endpoint also might skip a packet number intentionally, as suggested in [section 21.3](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-optimistic-ack-attack)?

Or maybe we should state in that section that skipping packet numbers too frequently would lead to an unnecessarily large number of ACK packets being sent in response.

Not that I have a strong opinion here, but I just wanted to make sure we'd be fine with what we'd have.

Reply:
I think it's worth mentioning something in 21.3, but I don't think it belongs in this definition.